.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
```

Comment:
@dnault 

This is the default configuration. There are [many configuration options ](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference) we have access to